help/compute/diff-studio.md

```diff
@@ -353,7 +376,42 @@ The export feature provides an extension of your project with [scripting](script
 * non-elementary and special functions' use
 * Datagrok packages' functions call
 
+## Solver management
+
+Manage the solver of ODEs to improve performance. Specify its settings in the `#meta.solver`-line:
+
+* the numerical method (`method`)
+* the maximum number of iterations (`maxIterations`)
+* the maximum computation time (`maxTimeMs`)
+
+Diff Studio implements the following [Rosenbrock–Wanner](https://doi.org/10.1016/j.cam.2015.03.010) methods for solving ODEs:
+
+|Method|Value|
+|-------------|--------|
+|The modified Rosenbrock triple|`'mrt'`|
+|The ROS3PRw method|`'ros3prw'`|
+|The ROS34PRw method|`'ros34prw'`|
+
+By default, Diff Studio uses ROS34PRw. You may specify the method as follows:
```

Comment:
Too long to read. Let's make one example with three options, instead.

Reply:
Reduced